tensorflow_serving/tools/docker/Dockerfile.devel-mkl

```diff
@@ -88,7 +88,9 @@ RUN mkdir /bazel && \
 
 # Download TF Serving sources (optionally at specific commit).
 WORKDIR /tensorflow-serving
-RUN curl -sSL --retry 5 https://github.com/tensorflow/serving/tarball/${TF_SERVING_VERSION_GIT_COMMIT} | tar --strip-components=1 -xzf -
```

Comment:
do you know why this does not work? is there a diff between the source trees, downloaded with these two different methods?

Reply:
I haven't seen a diff.
But like I said this is the only way I can get this Docker to build right now.
I'll look closer to see if I come across anything.